tests/lib/external/snapd-testing-tools/utils/log-parser

```diff
@@ -68,12 +34,12 @@ class Action:
 
     def __dict__(self):
```

Comment:
@zyga wrote:
I've left a remark in my commit message but we should not invent new dunder methods or change the behavior of existing methods. For as long as I remember dict is a special attribute that is a dictionary of the internals of the value. It is not a method and its content directly corresponds to the state of the object.

I've renamed this to _some_dict, please check if this is what you expected.

Reply:
done